common/src/main/java/io/netty/util/internal/PriorityQueue.java

```diff
@@ -34,4 +34,10 @@
      * @param node An object which is in this queue and the priority may have changed.
      */
     void priorityChanged(T node);
+
+    /**
+     * Removes all of the elements from this {@link PriorityQueue}, without resetting nodes. This should only be used
```

Comment:
`without resetting nodes` -> `without calling {@link PriorityQueueNode#priorityQueueIndex(DefaultPriorityQueue)} on any of the nodes.

Reply:
+1